src/components/datatables/components/TableHeader.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Can't you use the same TableFooter.js in here?

Reply:
If in future we want to add some items in header then we will have flexibility. and It will also improve readability of code.  Other then that it's same for now.

Reply:
If you are okay with adding this as a comment and remove this duplicate and add this back when we need more functionalities?